src/families/polkadot/js-getTransactionStatus.js

```diff
@@ -69,7 +71,24 @@ const getSendTransactionStatus = async (
     errors.amount = new AmountRequired();
   }
 
-  if (totalSpent.gt(a.spendableBalance)) {
+  const existentialDeposit = getExistentialDeposit(a);
+
+  if (
+    existentialDeposit.gt(0) &&
+    totalSpent.plus(existentialDeposit).gt(a.spendableBalance)
+  ) {
+    errors.amount = new NotEnoughSpendableBalance(null, {
+      minimumAmount: formatCurrencyUnit(
+        a.currency.units[0],
+        getExistentialDeposit(a),
```

Comment:
You could use the variable here since it's already computed

Reply:
nice catch yes